docs/UserGuide.md

```diff
@@ -233,15 +237,35 @@ Example(s):
 * `efilter CS2103T`
 * `efilter CS2100 Meeting`
 
+### Editing an entry
+
+Edits an existing entry with the specified index in Teaching Assistant.
+
+Format: `eedit INDEX [n/NAME] [sd/START_DATE] [ed/END_DATE] [t/TAG]...`
```

Comment:
should be id/INDEX
refer to the edit entry command usage if you don't trust me

this goes to edit contact as well because editing a contact by name is not possible